pkg/cri/sbserver/podsandbox/sandbox_delete.go

```diff
@@ -117,7 +118,7 @@ func (c *Controller) cleanupSandboxTask(ctx context.Context, sbCntr containerd.C
 		_, err = c.client.TaskService().Delete(ctx, &apitasks.DeleteTaskRequest{ContainerID: sbCntr.ID()})
 		if err != nil {
 			err = errdefs.FromGRPC(err)
-			if !errdefs.IsNotFound(err) {
+			if !errdefs.IsNotFound(err) && !strings.Contains(err.Error(), "container must be created") {
```

Comment:
Is this the common pattern for detecting errors in containerd? It seems like if this string were to change this will break again. Would it be better to break this into its own error type and check that instead?

Reply:
the error is defined here
```
// getInitProcess returns initial process
func (s *Service) getInitProcess() (process.Process, error) {
	s.mu.Lock()
	defer s.mu.Unlock()

	p := s.processes[s.id]
	if p == nil {
		return nil, errdefs.ToGRPCf(errdefs.ErrFailedPrecondition, "container must be created")
	}
	return p, nil
}
```
it is difficult to  change to own error type.